specs/v1.5/swagger-saral-apidoc.yaml

```diff
@@ -710,6 +710,11 @@ definitions:
             type: boolean
             description: "If this is set to true , app can operate in offline mode after frist navigation ."
             example: true
+          lock:
```

Comment:
@apurvabisht97, no need to have an additional flag in the login API. Just rely on lock document entries.